_rules/image-no-text-0va7u6.md

```diff
@@ -0,0 +1,204 @@
+- The specific presentation of the text rendered in the image resource can be achieved through formatted text. Otherwise, the rule might fail while [SC 1.4.5 Images of Text][sc1.4.5] and [SC 1.4.9 Images of Text (No Exception)][sc1.4.9] might be satisfied.
+
+## Accessibility Support
+
+_No accessibility support issues known._
+
+## Background
+
+This rule is designed specifically for [SC 1.4.5 Images of Text][sc1.4.5] which includes exceptions to the images it applies to that are not part of [SC 1.4.9 Images of Text (No Exception)][sc1.4.9]. Therefore, some images that are inapplicable for this rule can be applicable to [SC 1.4.9 Images of Text (No Exception)][sc1.4.9].
+
+- [Understanding Success Criterion 1.4.5: Images of Text][sc1.4.5]
+- [Understanding Success Criterion 1.4.9: Images of Text (No Exception)][sc1.4.9]
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This `img` element references an image resource that does not contain text.
+
+```html
+<img src="/test-assets/shared/fireworks.jpg" alt="fireworks going off behind the Eiffel tower at night" />
+```
+
+#### Passed Example 2
+
+This `input` element references an image resource that does not contain text.
+
+```html
+<input type="image" src="test-assets/shared/file.svg" alt="New file" />
+```
+
+#### Passed Example 3
+
+This `svg` element displays an image resource that does not contain text.
+
+```html
+<svg width="2in" height="3in" xmlns="http://www.w3.org/2000/svg">
+	<image x="20" y="20" width="200px" height="100px" href="/test-assets/shared/fireworks.jpg">
+		<title>Fireworks in Paris</title>
+	</image>
+</svg>
+```
+
+#### Passed Example 4
+
+This `object` element references an image resource where text is not the most significant content.
+
+```html
+<object data="/test-assets/0va7u6/times_square.jpg" title="Picture of Times Square, New York"></object>
+```
+
+#### Passed Example 5
+
+This `img` element references an image resource that contains text but where the presentation of the text is essential to convey the information.
+
+```html
+<p>The following image illustrates the use of cursive writing in the late nineteenth century.</p>
+<img src="/test-assets/0va7u6/letter.jpg" alt="Nineteenth century handwritten letter" />
+```
+
+#### Passed Example 6
+
+This `div` element has a background image with a logo with text. Logotypes are considered an essential exception.
+
+```html
+<div
+	role="img"
+	aria-label="W3C logo"
+	style="
+    width: 100px;
+    height: 100px;
+    background-image: url(test-assets/shared/w3c-logo.png);
+    background-repeat: no-repeat;
+  "
+></div>
+```
+
+#### Passed Example 7
+
+This `img` element is an image of text (the book covers), but it is just meant to decorate the webpage of a book store, therefore it is [purely decorative][].
+
+```html
+<img src="/test-assets/0va7u6/books.jpg" alt="" />
+<p>Welcome to my book store</p>
+```
```

Comment:
I suggest another passed example: button image that has letters, such as large and small "T" for changing text size, or an "F" for changing font type, etc. -- buttons we commonly find in editors and word processors but do not actually qualify as images of text even though they include text.

Reply:
Added